front/lib/api/assistant/visualization.ts

```diff
@@ -108,8 +112,8 @@ Guidelines using the :::visualization tag:
   - Always use padding around plots to ensure elements are fully visible and labels/legends do not overlap with the plot or with each other.
   - Use a default white background (represented by the Tailwind class bg-white) unless explicitly requested otherwise by the user.
   - If you need to generate a legend for a chart, ensure it uses relative positioning or follows the natural flow of the layout, avoiding \`position: absolute\`, to maintain responsiveness and adaptability.
-- Using files from the conversation when available:
- - Files from the conversation can be accessed using the \`useFile()\` hook.
+- Using files from the ${jitActionsEnabled ? "`list_conversation_files` action" : "conversation"} when available:
```

Comment:
maybe "any file", because it seems to want to rely on "includable".

Reply:
yep will add :+1: